packages/serverless-cdk-plugin/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@swarmion/serverless-cdk-plugin",
```

Comment:
Can you add the meta infos in the package.json? This is useful when publishing on npm for SEO and also enables users to quickly find the documentation. See `serverless-contracts` for examples.

In particular, it is mandatory to add 
```"publishConfig": {
    "access": "public"
  },
```
Otherwise npm will consider it as a private package